package.json

```diff
@@ -36,6 +36,7 @@
     "html-webpack-plugin": "^5.5.0",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^28.1.3",
+    "jest-environment-jsdom": "^29.3.1",
```

Comment:
Can you uninstall this for now? I think you can run `npm uninstall jest-environment-jsdom`

Reply:
Done!